azext_iot/tests/iothub/configurations/test_iot_config_int.py

```diff
@@ -294,6 +308,53 @@ def test_edge_deployments(self):
                 ],
             )
 
+            # Edge billable module deployment
```

Comment:
Comment should state something around creating a deployment with billable modules requires AAD auth.